### DISCUSSION TITLE
Write text size directly into Type Tool toolbar

### DISCUSSION THREAD

**Discussion by ghost** [opentoonz/opentoonz#6809] [Ideas]
**Is your feature request related to a problem? Please describe**

To avoid to transform texts manually to got the desired size that will be great if we can write it directly also.

Here the expected behavior about custom size:

![Custom size type tool](https://user-images.githubusercontent.com/29790512/94381291-2591ae00-0139-11eb-892e-3f2a8561183f.gif)

**Comment by RodneyBaker** [comment]
I don't think your request quite made it through @openanim 
Do you need assistance describing the feature?

**Comment by artisteacher** [comment]
I believe what @openanim means is that the type tool font sizes are severely limited. It's fine for the dropdown to stay as is, but we should be able to override those sizes by typing any size value for the font. That's how it works in Photoshop - so industry standard behavior.
<img width="113" alt="Screen Shot 2020-09-27 at 9 09 47 PM" src="https://user-images.githubusercontent.com/21159116/94381022-ce261a80-0105-11eb-9538-fd916ffe3bbc.png">
On vector levels, at least we can use the selection tool to correct the size without impacting the quality - though that shouldn't be a necessary step. On Toonz raster levels, resizing the text with the select tool can negatively impact the appearance.

**Comment by RodneyBaker** [comment]
Ah.  Thank you for that clarification.
Much appreciated.

**Comment by ghost** [comment]
@RodneyBaker I added a gif to my initial post to try to make this request clearer. Feel free to rectify if this brings more clarity. This request is exactly what @artisteacher explained in his post with the current weaknesses of the text tool.
